src/mudlet.cpp

```diff
@@ -1668,61 +1666,12 @@ Host * mudlet::getActiveHost()
         return 0;
 }
 
-
-
 void mudlet::addSubWindow( TConsole* pConsole )
 {
     mainPane->layout()->addWidget( pConsole );
     pConsole->show();//NOTE: this is important for Apple OSX otherwise the console isnt displayed
 }
 
-void mudlet::closeEvent(QCloseEvent *event)
```

Comment:
I think it would be an appropriate place for the new C++11 [override keyword](http://en.cppreference.com/w/cpp/language/override) yeah?